platform/darwin/src/MGLCircleStyleLayer.h

```diff
@@ -55,8 +55,8 @@ typedef NS_ENUM(NSUInteger, MGLCircleTranslateAnchor) {
     22: MGLStyleValue(rawValue: 180)
  ])
  layer.circleOpacity = MGLStyleValue(rawValue: 0.7)
- layer.predicate = NSPredicate(format: "%K == %@", "ethnicity", "ewok")
- mapView.style().add(layer)
+ layer.predicate = NSPredicate(format: "%K == %@", "marital-status", "married")
+ mapView.style.addLayer(layer)
```

Comment:
@1ec5 it's interesting that making the style a property instead of returning it in a method had this effect. Is this a case for `NS_SWIFT_NAME`?